api/service/explorer/service.go

```diff
@@ -102,15 +104,15 @@ func (s *Service) Run() *http.Server {
 
 	s.router = mux.NewRouter()
 	// Set up router for blocks.
-	s.router.Path("/blocks").Queries("from", "{[0-9]*?}", "to", "{[0-9]*?}").HandlerFunc(s.GetExplorerBlocks).Methods("GET")
+	s.router.Path("/blocks").Queries("from", "{[0-9]*?}", "to", "{[0-9]*?}", "page", "{[0-9]*?}", "offset", "{[0-9]*?}").HandlerFunc(s.GetExplorerBlocks).Methods("GET")
 	s.router.Path("/blocks").HandlerFunc(s.GetExplorerBlocks)
```

Comment:
how do we properly document all the api parameters and the usage in the code?  Please add sufficient comment so that ppl will understand the definition of each parameter and their usage. @johnwhitton